scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md

```diff
@@ -7,16 +7,19 @@ Complete (at least) through the training step (`GvsCreateFilterSet`) of the [GVS
 
 The wdl takes several inputs:
 
-**input_vcfs** - A collection of VCFS - generated by `GvsExtractCallSet`. These need not be subsetted down to the chromosome.
+**input_vcf_fofn** - A FOFN (file of file names) of output VCFs for control samples generated by `GvsExtractCallSet`. These need not be subsetted down to the chromosome.
```

Comment:
doesn't our extract automatically split by chr?

Reply:
I think that's right. But the P+S logic doesn't seem to require that it receive VCFs containing only data for the chromosomes on which it's trying to calculate P+S.